submit.go

```diff
@@ -71,6 +71,7 @@ type (
 		JobConfig              *string  `arg:"--job-conf"`
 		File                   string   `json:"file" ini:"file" arg:"positional"`
 		Parameters             []string `arg:"positional"`
+		NotCompletedExitCode   int64    `arg:"--not-completed-exit-code" help:"Exit code for TERMINATED and FAILED job, default is 0"`
```

Comment:
Hey @Gudsfile 
seems like `--not-completed` sentence not appropriate,
as `TERMINATED` and `FAILED` are considered interrupted 

 what do you think of this suggestion ?

```suggestion
		NotCompletedExitCode   int64    `arg:"--interrupted-exit-code" help:"Exit code for TERMINATED and FAILED job, default is 0"`
```

Reply:
Ty for the review. I've also modified the help message, variables and unit tests accordingly.